website/docs/concepts/scopes.mdx

```diff
@@ -0,0 +1,262 @@
+---
+title: Scopes
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+Scoping in Riverpod is an important thing to understand.
+
+Many issues that developers have with Riverpod come from misunderstanding scope.
+
+## Container Scope
```

Comment:
I don't understand what this part is about

Reply:
Just this part, or this whole page? The intent of this part is to educate developers why they shouldn't use ProviderContainers in flutter (except with UncontrolledProviderScope), and why provider definitions can be global even though their state is not. I feel like it is a major source of confusion for newcomers.